concordium-smart-contract-testing/src/types.rs

```diff
@@ -600,3 +632,53 @@ pub struct ExchangeRateError;
 #[derive(Debug, Error)]
 #[error("Any signer must have at least one key.")]
 pub struct ZeroKeysError;
+
+/// Errors that occur while setting up the connection to an external node.
+#[derive(Debug, thiserror::Error, PartialEq, Eq)]
+pub enum SetupExternalNodeError {
+    /// It was not possible to connect to a node on the provided endpoint.
+    #[error("Could not connect to the provided endpoint.")]
+    CannotConnect,
```

Comment:
Why does this not have an inner error that would tell why the connection could not be established?